### PR TITLE
Redesign & refactor

### PR DESCRIPTION
**Pull request by saharmor**

### CODE REVIEW

**Comment by saharmor** [comment]
Note: need to make sure the Github page + notebook are up to date after merge

File: front/README.md [hunk 1]
```diff
@@ -1,5 +1,25 @@
-## Dalle-Service front
+# react-super-cmd example ⚡
```
Comment (afiaka87) [inline front/README.md:1]: Looks like you've still got some boilerplate leftover here.

Comment (saharmor) [reply front/README.md]: Yep, removed.

**Comment by saharmor** [comment]
@afiaka87 any news on this one? :)

**Comment by rom1504** [comment]
hi,
I like some of the improvements, but I am not super convinced by the complexity introduced by moving to react.
Also not super convinced by the look of the new UI putting all fields at the same level although the important one is only the text input bar.

**Comment by saharmor** [comment]
I'd suggest removing the `backend url` and `selected model` altogether and move them into a configuration json in the frontend. UI would be:
Screenshot 2021-07-08 at 10 56 01

Thoughts?

**Comment by rom1504** [comment]
no both options definitely need to be shown to the user as there is no fixed backend hosting system. The front is hosted on github, the back is hosted by users. I just think they should not be as big as the text bar

**Comment by afiaka87** [comment]
> @afiaka87 any news on this one? :)

Entirely up to @rom1504

**Comment by saharmor** [comment]
> no both options definitely need to be shown to the user as there is no fixed backend hosting system. The front is hosted on github, the back is hosted by users. I just think they should not be as big as the text bar

@rom1504 👍 pushed a fix

**Comment by rom1504** [comment]
hi,
I realize this must have taken quite some amount of work, but I am not convinced by the use of react in this project that I maintain.
what do you think about putting this in another repo so we have an alternative UI for dalle ?

**Pull request closed without merging**